quant.module

```diff
@@ -686,6 +734,116 @@ function quant_log($message, $replacements = array(), $level = WATCHDOG_INFO) {
   watchdog('quant', $message, $replacements, $level);
 }
 
+/**
+ * Log seed data to database.
+ */
+function quant_log_seed($context, $message = 'Successful.') {
+  if (!isset($context['url'])) {
+    quant_log('No URL provided for quant_log_seed.', [], WATCHDOG_ERROR);
+    return;
+  }
+
+  $md5 = 0;
+  if (isset($context['type']) && isset($context['location']) && $context['type'] == 'file') {
+    $md5 = isset($context['md5']) ? $context['md5'] : md5_file($context['location']);
```

Comment:
We may need to look at this when `quant_alternate_file_handler` is enabled.

In this mode files may be stored externally (e.g S3/other storage backends). The file is first downloaded locally to disk before md5 hash comparison, so we just need to check this logic has already fired and `$context['location']` contains the path on disk to the temporary file.

In this mode it is more desirable to rely on the updated timestamp _before_ md5 comparison, so we don't bother downloading files if it's not necessary.

Reply:
Taking a look at this now.